scripts/populate_new_and_noteworthy_projects.py

```diff
@@ -0,0 +1,137 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.project.model import Pointer
+from website.settings import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE, QA_USER_IDS, URL_BASE as BASE
+
+logger = logging.getLogger(__name__)
+
+def retrieve_data(url):
+    """ Fetch data and decode json """
+    response = requests.get(url)
+    data = response.json()
+    return data
+
+def get_api_base_path():
```

Comment:
Again, why do we need this given that we `settings.DOMAIN` and `settings.API_DOMAIN`? This code is fragile and hard to maintain; we would have to modify these functions every time we make a new deployment (e.g. when we create staging3).